### PR TITLE
Veoss 554

### PR DESCRIPTION
**Pull request by oliwiaKot**
Added remove functionality for correction countries and periods, check loop mode will be done as part of VEOSS-599 as the journey is modified there by adding another mini cya

### CODE REVIEW

File: app/config/Module.scala [hunk 1]
```diff
@@ -25,7 +25,12 @@ class Module extends AbstractModule {
 
   override def configure(): Unit = {
     bind(classOf[DataRequiredAction]).to(classOf[DataRequiredActionImpl]).asEagerSingleton()
-    bind(classOf[Clock]).toInstance(Clock.systemDefaultZone.withZone(ZoneOffset.UTC))
+    bind(classOf[Clock]).toInstance(
+      Clock.fixed(
+        Instant.parse("2023-10-01T12:00:00.00Z"),
+        ZoneId.of("Australia/Melbourne")
+      )
+    )
```
Comment (DevAndyroo) [inline app/config/Module.scala:33]: revert the clock please

File: app/controllers/corrections/UndeclaredCountryCorrectionController.scala [hunk 2]
```diff
@@ -67,7 +67,6 @@ class UndeclaredCountryCorrectionController @Inject()(
             value =>
               for {
                 updatedAnswers <- Future.fromTry(request.userAnswers.set(UndeclaredCountryCorrectionPage(periodIndex, countryIndex), value))
-                _ <- cc.sessionRepository.set(updatedAnswers)
```
Comment (DevAndyroo) [inline app/controllers/corrections/UndeclaredCountryCorrectionController.scala:70]: why does this get removed?

**Review by DevAndyroo — CHANGES_REQUESTED** [review]

File: app/controllers/corrections/UndeclaredCountryCorrectionController.scala (hunk 2 above)
Comment (warrenskiwork1979) [reply app/controllers/corrections/UndeclaredCountryCorrectionController.scala:70]: We don't think we need this saved, it's not used from what we've seen and just made it easier for cleanup. Unless you feel we should be saving this?

File: app/config/Module.scala (hunk 1 above)
Comment (warrenskiwork1979) [reply app/config/Module.scala:33]: This has slipped back in when trying to resolve a new GPG key issue with rebase -i. Will remove.

**Review by DevAndyroo — APPROVED** [review]

**Pull request merged**